app/models/status.rb

```diff
@@ -535,4 +538,14 @@ def decrement_counter_caches
     reblog&.decrement_count!(:reblogs_count) if reblog?
     thread&.decrement_count!(:replies_count) if in_reply_to_id.present? && distributable?
   end
+
+  def trigger_create_webhooks
+    TriggerWebhookWorker.perform_async('status.created', 'Status', id) if local?
+    TriggerWebhookWorker.perform_async('status.created.distributable', 'Status', id) if local? && distributable?
```

Comment:
This concept gives me a bit of pause... It'd be easier for me to accept the PR without it. If we decide to change this naming scheme or how this works later it'll be a bother so I would prefer not to hastily add it.

Reply:
Sure, I can split this out and resubmit that half when we have a better name for it.

Reply:
@Gargron Done.